### PR TITLE
DTM and radio test build empty_app_core sample by default as child image.

### PR DESCRIPTION
**Pull request by KAGA164**
This PR adds possibility to build empty_app_core by default for the Direct Test Mode sample and the Radio Test sample

### CODE REVIEW

File: samples/bluetooth/direct_test_mode/README.rst [hunk 1]
```diff
@@ -210,6 +210,8 @@ Building and running
    It does not require any counterpart application sample.
    However, you must still program the application core to boot up the network core.
    You can use any sample for this, for example, the :ref:`nrf5340_empty_app_core`.
+   The :ref:`nrf5340_empty_app_core` is built and flashed automatically by default.
+   If you want to program another sample for the application core, you must unset :option:'CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE'.
```
Comment (peknis) [inline samples/bluetooth/direct_test_mode/README.rst:214]:
```suggestion
   The :ref:`nrf5340_empty_app_core` is built and programmed automatically by default.
   If you want to program another sample for the application core, unset the :option:'CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE' option.
```

File: samples/peripheral/radio_test/README.rst [hunk 2]
```diff
@@ -137,6 +137,8 @@ Building and running
    On the |nRF5340DKnoref|, the Radio test sample is a standalone network sample that does not require any counterpart application sample.
    However, you must still program the application core to boot up the network core.
    You can use any sample for this, for example :ref:`nrf5340_empty_app_core`.
+   The :ref:`nrf5340_empty_app_core` is built and flashed automatically by default.
+   If you want to program another sample for the application core, you must unset :option:'CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE'.
```
Comment (peknis) [inline samples/peripheral/radio_test/README.rst:141]: Needs same fixes as above.

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by qung — APPROVED** [review]

**Review by kapi-no — APPROVED** [review]

File: samples/bluetooth/direct_test_mode/README.rst (hunk 1 above)
Comment (KAGA164) [reply samples/bluetooth/direct_test_mode/README.rst]: Done

File: samples/peripheral/radio_test/README.rst (hunk 2 above)
Comment (KAGA164) [reply samples/peripheral/radio_test/README.rst]: Fixed

**Review by peknis — APPROVED** [review]

**Review by grochu — APPROVED** [review]

**Pull request merged**